README.md

```diff
@@ -18,6 +18,16 @@ A Secret of Mana Randomizer bot for Discord.
 - .Net Core: https://dotnet.microsoft.com/download
 - DSharpPlus: https://github.com/DSharpPlus/DSharpPlus
 
+### Permissions
+
+For all commands to work, the following permissions must be given to the bot on discord
+
+- Send Messages
+- Manage Messages
+- Manage Roles
```

Comment:
Maybe note that the bot only needs to manage the role assigned in the config.
So if you move the bot role down enough, it shouldn't be able to give, add or remove roles that could compromise the security of the server.

Reply:
Added some details in README in 0a6d5cf56dd2ab174f54457026a7e5306d681edb.